experimenter/experimenter/outcomes/tests/test_outcomes.py

```diff
@@ -83,6 +94,17 @@ def test_load_outcomes_by_application(self):
                 ),
                 desktop_outcomes,
             )
+        self.assertIn(
+            Outcome(
+                application=NimbusExperiment.Application.DESKTOP,
+                description="a test outcome missing metrics",
+                friendly_name="test outcome",
+                slug="missing_metrics",
+                is_default=False,
+                metrics=[],
```

Comment:
do we have a test where metrics is not default `[]`?

Reply:
The other tests and fixtures cover the cases where metrics are present.